notebooks/FugueSQL.ipynb

```diff
@@ -0,0 +1,510 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "f39e2dbc-21a1-4d9a-bed7-e2bf2bd25bb8",
+   "metadata": {},
+   "source": [
+    "# Fugue + Dask-sql\n",
+    "\n",
+    "When `dask-sql` is installed, Fugue can automatically detect and set it as the default Dask execution engine."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a35d98e6-f24e-46c4-a4e6-b64d649d8ba7",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from dask.distributed import Client\n",
+    "\n",
+    "client = Client(processes=True)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ff633572-ad08-4de1-8678-a8fbd09effd1",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>a</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>xyz</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "     a\n",
+       "0  xyz"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<small>schema: a:str</small>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%%fsql dask\n",
```

Comment:
This cell magic needs to be registered manually to work correctly, right? If so, should we add a block like

```python
from fugue_notebook import setup

setup()
```

Preceding this and explain what's happening there? At least on my end, this notebook doesn't work without running that block.

Reply:
This is another big change in 0.7.0, where we start deprecating `fugue_notebook`, instead, it's separated into [fugue-jupyter](https://pypi.org/project/fugue-jupyter/), I will update the notebook. Great catch, thanks!